README.md

```diff
@@ -1,13 +1,35 @@
+Klaytn Docs introduces [Klaytn](https://klaytn.foundation/), the blockchain of choice for web3 era with high performance and great user experience. In these documents you will find detailed explanations on Klaytn along with guidelines and instructions on how to use Klaytn and interact with the Klaytn network. More information on how to use Klaytn Docs is depicted on the [Klaytn Docs Wiki](https://github.com/klaytn/klaytn-docs/wiki)
 
-- If you want to change the source content, you should update the `master` branch of `klaytn-docs`.
-- If you want to change the translation, you should work on [Crowdin](https://crowdin.com/project/klaytn-docs). Then the modified translation will be pushed back to the `l10n` branch of `klaytn-docs`. To request access to the translation, follow the link - https://crwd.in/klaytn-docs. 
-- Do NOT push to `klaytn-docs:l10n` directly. It will be overwritten by Crowdin and the change will get lost.
+## Want to Contribute to the Translation? <a id="want-to-translate"></a>
 
-`dev` branch of this repo mirrors the `ko/docs` subfolder of the `klaytn-docs:l10n` periodically. Approved contents will be merged to `master` branch and published at https://ko.docs.klaytn.com. 
+Thank you for your interest in Klaytn Docs! As an open source project, Klaytn Docs is open for contribution by anyone, which means you can also participate in the translation of the English version into other languages.
 
+However, **you are not supposed to push to this repo directly.** 
+
+Instead, you can contribute to the translation on the [Crowdin](https://crowdin.com/project/klaytn-docs) platform.
+
+**Workflow summary**
+
+- Submitted translations get approved by our maintainers.
+- The approved translations will be pushed to the `l10n` branch of `klaytn-docs`. The `dev` branch of this repo mirrors the `ko/docs` subfolder of `klaytn-docs:l10n` periodically. 
+- Our maintainers will merge the changes to the `master` branch, which will be reflected at https://ko.docs.klaytn.com. 
+
+Note: **Do NOT push to `klaytn-docs:l10n` directly.** It will be overwritten by Crowdin and the changes will get lost.
+
+To start translating, follow the [link](https://crowdin.com/project/klaytn-docs/invite?d=85n6j4j675t683c3h463e3t6c3q4r48383d3)
+
+## Want to Contribute to Klaytn Docs? <a id="want-to-contribute"></a>
+
+You can also contribute to the source text in English in [Klaytn Docs](https://github.com/klaytn/klaytn-docs).
+
+You can do that by creating a pull request on GitHub. For more details, please refer to [Contribution Guide](https://github.com/klaytn/klaytn-docs/wiki#contributing).
+
+## Need Help? <a href="#need-help" id="need-help"></a>
+
+If you have any questions, please visit our [Gitter channel](https://gitter.im/klaytn/klaytn-docs?utm_source=share-link&utm_medium=link&utm_campaign=share-link), [Klaytn Developers Forum](https://forum.klaytn.com/) and [Discord channel](https://discord.gg/mWsHFqN5Zf).
```

Comment:
@neoofklaytn Do we have any analytics connected with gitter? It seems we have utm tags in the link.

Reply:
@kjhman21 This is the standard link with pre-configured utm tags, from Gitter. We have not had analytics plugin integrated to the channels and there are plug-in options we could look to integrate on Gitter. Thanks.